app/src/Services/modules/events/index.ts

```diff
@@ -20,6 +20,12 @@ export interface Event {
   video_file: string
   video_thumbnail: string
   images: string[]
+  speakers: EventSpeaker[]
+}
+
+export interface EventSpeaker {
+  name: string
+  url: string
```

Comment:
`addedToCalendar` doesn't belong as part of an `EventSpeaker` -- it was a property of the `Event` interface and should be moved back up there